packages/caver-account/src/accountKey/accountKeyWeightedMultiSig.js

```diff
@@ -0,0 +1,117 @@
+/*
+    Copyright 2020 The caver-js Authors
+    This file is part of the caver-js library.
+
+    The caver-js library is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    The caver-js library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with the caver-js. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+const RLP = require('eth-lib/lib/rlp')
+const Bytes = require('eth-lib/lib/bytes')
+const WeightedPublicKey = require('./weightedPublicKey')
+const utils = require('../../../caver-utils')
+const { ACCOUNT_KEY_TAG } = require('./accountKeyHelper')
+
+/**
+ * Representing an AccountKeyWeightedMultiSig.
+ * @class
+ */
+class AccountKeyWeightedMultiSig {
+    /**
+     * Decodes an RLP-encoded AccountKeyWeightedMultiSig string.
+     * @param {string} rlpEncodedKey - An RLP-encoded AccountKeyWeightedMultiSig string.
+     * @return {AccountKeyWeightedMultiSig}
+     */
+    static decode(rlpEncodedKey) {
+        rlpEncodedKey = utils.addHexPrefix(rlpEncodedKey)
+        if (!rlpEncodedKey.startsWith(ACCOUNT_KEY_TAG.ACCOUNT_KEY_WEIGHTED_MULTISIG_TAG))
+            throw new Error(`Cannot decode to AccountKeyWeightedMultiSig: ${rlpEncodedKey}`)
```

Comment:
```suggestion
            throw new Error(`Cannot decode to AccountKeyWeightedMultiSig. The prefix must be `+ACCOUNT_KEY_TAG.ACCOUNT_KEY_WEIGHTED_MULTISIG_TAG+`: ${rlpEncodedKey}`)
```

Please correct the syntax if it's wrong. What I wanted to do is to make sure to know the exact reason of the failure.

Reply:
Okay i edited error message!